test/pseudo-tty/no_dropped_stdio.js

```diff
@@ -0,0 +1,13 @@
+// https://github.com/nodejs/node/issues/6456#issuecomment-219320599
+// https://gist.github.com/isaacs/1495b91ec66b21d30b10572d72ad2cdd
+'use strict';
+require('../common');
+// 1025 bytes of characters, just enough to overflow the 1kb OS X TTY buffer.
+const bytes = 1024 - 37;
+
+var out = new Array(bytes).join('o');
+out += 'O'; // End ID character
+out = out.replace(/(o{50})/g, '$1\n');
```

Comment:
This results in 1007 bytes which doesn't really do the trick. Can I suggest changing the above 5 lines to:

```
// create 1052 bytes of output, wrapped at 50 columns, terminated with an 'O'
// enough to overflow the 1kb OS X TTY buffer
var out = new Array(22).join(new Array(50).join('o') + '\n') + 'O\n'
```

Then the no_dropped_stdio.out would need to become:

```
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
ooooooooooooooooooooooooooooooooooooooooooooooooo
O
```